### ISSUE TITLE
execute the created code

### ISSUE THREAD

**Issue by MajeedKazemi** [MajeedKazemi/blocks-to-code-old#4]
convert everything to JS and execute JS locally. This is a better approach compared to using Pyodide as it will allow us to do more interesting things such as getting user input through a modal similar to Scratch, or display output in interesting ways.

Using JS would also allow future versions of the tool to create interactive outputs (draw lines, animate objects, and etc.)

**Comment by user-74** [comment]
Agreed, javascript execution is preferred over python, and we can equate logging with print/say statements.

The following is an example of how we can do this.

```js
var test = "console.log("Hello World");"

function execute(code) {
  var F = new Function(code);
  return (F());
}

execute(test);
```

**Comment by MajeedKazemi** [comment]
use prompt to get input from user and test it
